src/components/DatePicker/__stories__/datepicker.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import moment from "moment";
+import DatePickerComponent from "../DatePicker";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate, createStoryMetaSettings } from "../../../storybook";
+import {
+  TEXT_FIELD,
+  DROPDOWN,
+  DIALOG_CONTENT_CONTAINER
+} from "../../../storybook/components/related-components/component-description-map";
+import "./DatePicker.stories.scss";
+import DialogContentContainer from "../../DialogContentContainer/DialogContentContainer";
+
+export const metaSettings = createStoryMetaSettings({
+  component: DatePicker
+});
+
+export const DatePicker = props => {
```

Comment:
We trying really hard not to declare the component outside of a story because this way the users which read the story can't see how we implement the component inside the story and it's make the story not helpful for them. So I understand it will create a code which repeat itself, but still let's create a different implementation for each story. We can reuse classnames if it helps.